README.md

```diff
@@ -200,6 +200,15 @@ const res = utils.try(function () {
 // {error: Error, value: undefined}
 ```
 
+### argumentsToArray
+
+```js
+function() {
+  const arr = utility.argumentsToArray(arguments);
```

Comment:
果然是不行的，只要传递了 arguments 就是无法优化.

无法优化: `[aborted optimizing 0x2a511c0869c9 <JS Function testFn (SharedFunctionInfo 0x2df6d193d9b1)> because: Bad value context for arguments value]`

```js
function testFn() {
  var res = argumentsToArray(arguments);
  return res.length;
}
```